keyboards/bt66tech/bt66tech60/rules.mk

```diff
@@ -0,0 +1,37 @@
+# MCU name
+MCU = STM32F103
+
+# Bootloader selection
+BOOTLOADER = stm32duino
+
+
+
+
+
+
+
+
+
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = yes      # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+WS2812_DRIVER = spi
+DEFAULT_FOLDER = handwired/bt66tech/bt66tech60
+
```

Comment:
```suggestion

DEFAULT_FOLDER = handwired/bt66tech/bt66tech60
```

Reply:
add LAYOUTS = 60_ansi in rules.mk make my pull request check error , 
are you sure about this?
if you sure can you tell me how fix git check error after add LAYOUTS = 60_ansi

Reply:
Your layout macro and info.json look correct, so most likely it is just some community layout failing to build. You can safely ignore it.